ledger.ipldsch

```diff
@@ -20,6 +20,8 @@ type Subset struct {
   last   Int
   # The list of blocks in this subset.
   blocks [ Link ] # [ &Block ]
+  # Block metadata
+  blocks_metadata [ Link ] # [& BlockMeta]
```

Comment:
Is subset the correct place where we split it out? Or should there be a `SubsetMeta` object under Epoch?